webapp/components/generic/SearchableInput/SearchableInput.spec.js

```diff
@@ -32,81 +32,79 @@ describe('SearchableInput.vue', () => {
   }
 
   describe('mount', () => {
-    describe('testing custom functions', () => {
-      let select
+    let select
 
-      beforeEach(() => {
-        select = wrapper.find('.ds-select')
-        select.trigger('focus')
-        select.element.value = 'abcd'
-      })
+    beforeEach(() => {
+      select = wrapper.find('.ds-select')
+      select.trigger('focus')
+      select.element.value = 'abcd'
+    })
 
-      it('opens the select dropdown when focused on', () => {
-        expect(wrapper.find('.is-open').exists()).toBe(true)
-      })
+    it('opens the dropdown when focused', () => {
+      expect(wrapper.find('.ds-select-dropdown').isVisible()).toBe(true)
+    })
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-15T16:20:21Z" title="Wednesday, January 15th 2020, 5:20:21 pm +01:00">Jan 15, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -32,81 +32,79 @@ describe('SearchableInput.vue', () => {
   }
 
   describe('mount', () => {
-    describe('testing custom functions', () => {
-      let select
+    let select
 
-      beforeEach(() => {
-        select = wrapper.find('.ds-select')
-        select.trigger('focus')
-        select.element.value = 'abcd'
-      })
+    beforeEach(() => {
+      select = wrapper.find('.ds-select')
+      select.trigger('focus')
+      select.element.value = 'abcd'
+    })
 
-      it('opens the select dropdown when focused on', () => {
-        expect(wrapper.find('.is-open').exists()).toBe(true)
-      })
+    it('opens the dropdown when focused', () => {
+      expect(wrapper.find('.ds-select-dropdown').isVisible()).toBe(true)
+    })
```


this is an improvement, for sure in my mind...